From d2a127f59c30db0e7fd58eb0286df8451d06d2c8 Mon Sep 17 00:00:00 2001
From: Ash Charles <ashcharles@gmail.com>
Date: Wed, 4 Jun 2014 12:30:52 -0700
Subject: [PATCH 12/14] Change files compiled for drm.ko

Two things:
 1. Most of the files for drm are included in the kernel---these
    need to be manually copied over to this package for a SUPPORT_XORG=1
    build.  It is not clear that the drm.ko provided by the is not
    sufficient.
 2. drm_procs.c went along with the rest of the procfs stuff.  See
      commit cb6458f97b53d7f73043206c18014b3ca63ac345
      Author: Daniel Vetter <daniel.vetter@ffwll.ch>
      Date:   Thu Aug 8 15:41:34 2013 +0200

Signed-off-by: Ash Charles <ashcharles@gmail.com>
---
 services4/3rdparty/linux_drm/Kbuild | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/services4/3rdparty/linux_drm/Kbuild b/services4/3rdparty/linux_drm/Kbuild
index d01ef39..c4a02b3 100644
--- a/services4/3rdparty/linux_drm/Kbuild
+++ b/services4/3rdparty/linux_drm/Kbuild
@@ -33,31 +33,31 @@ drm-y := pvr_drm_stubs.o drm_auth.o drm_bufs.o drm_cache.o drm_context.o drm_dma
 	drm_sysfs.o drm_hashtab.o drm_sman.o drm_mm.o \
 	drm_crtc.o drm_modes.o drm_edid.o \
 	drm_info.o drm_debugfs.o drm_encoder_slave.o
-else
-# Works for 2.6.37 till 3.2 kernel
-drm-y := pvr_drm_stubs.o drm_auth.o drm_bufs.o drm_cache.o drm_context.o drm_dma.o drm_drv.o drm_fops.o drm_gem.o drm_ioctl.o drm_irq.o \
-	drm_lock.o drm_memory.o drm_proc.o drm_stub.o drm_vm.o \
-	drm_agpsupport.o drm_scatter.o ati_pcigart.o drm_pci.o \
-	drm_sysfs.o drm_hashtab.o drm_sman.o drm_mm.o \
-	drm_crtc.o drm_modes.o drm_edid.o \
-	drm_info.o drm_debugfs.o drm_encoder_slave.o drm_global.o drm_platform.o drm_trace_points.o
-endif
-
-# For 3.3 kernel only
+#else
+## Works for 2.6.37 till 3.2 kernel
 #drm-y := pvr_drm_stubs.o drm_auth.o drm_bufs.o drm_cache.o drm_context.o drm_dma.o drm_drv.o drm_fops.o drm_gem.o drm_ioctl.o drm_irq.o \
 #	drm_lock.o drm_memory.o drm_proc.o drm_stub.o drm_vm.o \
 #	drm_agpsupport.o drm_scatter.o ati_pcigart.o drm_pci.o \
-#	drm_sysfs.o drm_hashtab.o drm_mm.o \
+#	drm_sysfs.o drm_hashtab.o drm_sman.o drm_mm.o \
 #	drm_crtc.o drm_modes.o drm_edid.o \
 #	drm_info.o drm_debugfs.o drm_encoder_slave.o drm_global.o drm_platform.o drm_trace_points.o
+endif
 
-# For greater than/equal to 3.4 till 3.8 kernel
+# For 3.3 kernel only
 #drm-y := pvr_drm_stubs.o drm_auth.o drm_bufs.o drm_cache.o drm_context.o drm_dma.o drm_drv.o drm_fops.o drm_gem.o drm_ioctl.o drm_irq.o \
 #	drm_lock.o drm_memory.o drm_proc.o drm_stub.o drm_vm.o \
 #	drm_agpsupport.o drm_scatter.o ati_pcigart.o drm_pci.o \
 #	drm_sysfs.o drm_hashtab.o drm_mm.o \
 #	drm_crtc.o drm_modes.o drm_edid.o \
-#	drm_info.o drm_debugfs.o drm_encoder_slave.o drm_global.o drm_platform.o drm_trace_points.o drm_prime.o
+#	drm_info.o drm_debugfs.o drm_encoder_slave.o drm_global.o drm_platform.o drm_trace_points.o
+
+# For greater than/equal to 3.4 till 3.8 kernel
+drm-y := pvr_drm_stubs.o drm_auth.o drm_bufs.o drm_cache.o drm_context.o drm_dma.o drm_drv.o drm_fops.o drm_gem.o drm_ioctl.o drm_irq.o \
+	drm_lock.o drm_memory.o drm_stub.o drm_vm.o \
+	drm_agpsupport.o drm_scatter.o ati_pcigart.o drm_pci.o \
+	drm_sysfs.o drm_hashtab.o drm_mm.o \
+	drm_crtc.o drm_modes.o drm_edid.o \
+	drm_info.o drm_debugfs.o drm_encoder_slave.o drm_global.o drm_platform.o drm_trace_points.o drm_prime.o
 
 # less than 2.6.32 kernel
 
-- 
1.8.3.2

